ex6/ex6.txt

```diff
@@ -0,0 +1,6 @@
+1. Docker can reuse previously built layers during image builds. layers are fetched from the cache, reducing the time required to build the image.
+
+2. I think the second option is better because it could cache more commands and save image building time later. 
```

Comment:
we have to remember that each command is a new layer- so more cache is saved, and the image uses more space. We have to remember it when creating the dockerfile so in order to maybe save time in the future we will create a huge file.